### PR TITLE
Add ability to run without authz.

### PR DESCRIPTION
**Pull request by jimwsplk**

### CODE REVIEW

**Comment by jimwsplk** [comment]
These are the kernel-collector changes to allow it to run and connect to the OpenTelemetry collector or the Flowmill server without using authz.

File: collector/kernel/kernel_collector.cc [hunk 1]
```diff
@@ -354,18 +359,31 @@ void KernelCollector::send_connection_metadata()
   struct struct_name __##struct_name##__##buf_name2;                           \
   char buf_name2[sizeof(__##struct_name##__##buf_name2.field2)] = {};
 
-  auto const &token = authz_token_.payload();
-  LOG::info(
+  switch(intake_config_.auth_method()) {
+  case collector::AuthMethod::none:
+    if (intake_config_.encoder() == IntakeEncoder::binary) {
+      writer_.no_auth_connect(static_cast<u8>(ClientType::kernel), jb_blob{host_info_.hostname});
```
[the remark this answers is not in the document]
Comment (bjandras) [reply collector/kernel/kernel_collector.cc:365]: Make sure to add the message to flowmill.render file.

[the remark this answers is not in the document]
Comment (jimwsplk) [reply collector/kernel/kernel_collector.cc:365]: I was wondering about that.  I made the changes to the GitLab flowmill repo render/flowmill.render.  Do I need to just make those same changes in the GitLab repo flowmill-collector/render/flowmill.render?

**Comment by samiura** [comment]
LGTM

**Review by samiura — APPROVED** [review]

**Pull request merged**